### PR TITLE
Tidy up licenses

### PR DESCRIPTION
**Pull request by newhoggy**
I moved the date range part of the PR to a separate PR. https://github.com/input-output-hk/cardano-node/pull/4755

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Input Output (Hong Kong) Ltd.
+   Copyright 2021 Input Output (Hong Kong) Ltd.
```
Comment (coot) [inline LICENSE:190]:
```suggestion
   Copyright 2021-2023 Input Output (Hong Kong) Ltd.
```

Comment (coot) [reply LICENSE:190]: The same in other places, or am I missing something?

Comment (newhoggy) [reply LICENSE]: I didn't realise there was a date in the `LICENSE` files.  I also found that the files sometimes disagreed, in which case I took the older date.

Comment (JaredCorduan) [reply LICENSE]: I am pretty sure that the "APPENDIX: How to apply the Apache License to your work." section of the `LICENSE` is not supposed to be there (it looks to be an explanation for how to make the `NOTICE` file).

Comment (newhoggy) [reply LICENSE]: I have removed the appendix!

**Comment by newhoggy** [comment]
I moved the date range part of the PR to a separate PR.

**Pull request closed without merging**